lib/file-info-view.coffee

```diff
@@ -10,10 +12,30 @@ class FileInfoView extends HTMLElement
     @bufferModified.classList.add('buffer-modified')
     @appendChild(@bufferModified)
 
+    @newTooltip()
+
     @activeItemSubscription = atom.workspace.onDidChangeActivePaneItem =>
       @subscribeToActiveItem()
     @subscribeToActiveItem()
 
+    clickHandler = ->
+      atom.clipboard.write(@getActiveItem()?.getPath?())
+      setTimeout =>
+        @newTooltip()
+      , 2000
+
+    @addEventListener('click', clickHandler)
+    @clickSubscription = new Disposable => @removeEventListener('click', clickHandler)
+
+  newTooltip: ->
```

Comment:
Yes, quite possibly. I wanted to represent the fact that, upon pane changes or after having been clicked already, it will *replace* whatever tooltip was there before, so it doesn't just create one.

I'll see what I can find out.

But I'll see if I can find